src/plugins/irc-events/link.js

```diff
@@ -93,13 +93,8 @@ function fetch(url, cb) {
 		return;
 	}
 	var length = 0;
-	var limit = 1024 * 10;
+	var limit = Helper.config.prefetchMaxImageSize * 1024;
```

Comment:
In default config, this change will fetch up to 512kb of HTML content instead of the 10kb. Can we keep the `"response"` callback, but instead of aborting the request, reduce the `limit`, so that we still only fetch 10kb of HTML, and `prefetchMaxImageSize` of unknown content?